lib/src/command/dependency_services.dart

```diff
@@ -200,6 +98,31 @@ class DependencyServicesReportCommand extends PubCommand {
         singleBreakingVersion = singleBreakingPackagesResult
             ?.firstWhereOrNull((element) => element.name == package.name);
       }
+      PackageId? smallestUpgrade;
+      final disallowedRanges = disallowed
+          .firstWhereOrNull((disallowed) => disallowed.ref == package.toRef());
+      if (disallowedRanges != null &&
+          !disallowedRanges.constraints.every(
+            (element) => !element.constraint.allows(package.version),
+          )) {
```

Comment:
Instead of not every not, could we do: `any`?

Reply:
Done (ish)